### DISCUSSION TITLE
Add or Inject a Class to actual SB Template?

### DISCUSSION THREAD

**Discussion by Riskbreaker01** [storybookjs/storybook#29560] [Help]
### Summary

IS it possible to add a unique class to specific stories so a unique css can only target those templates?

Im able to add a decorator but it only adds it to the canvas/layout but not the SB template itself

<img width="643" alt="Screenshot 2024-11-06 at 4 29 42 PM" src="https://github.com/user-attachments/assets/d7cfad3f-3c08-4e89-a103-78a942a28440">

Would like to add a unique class to it like "template-b":

`class=" sbdocs sbdocs-preview sb-unstyled css-hd7ysc template-b"`

**Comment by smlimasu** [comment]
You can select it using the :has() selector, like .sb-docs:has(.template-b). If you follow the way with the decorators for example.

If it's about the docs page you can provide your own docs templates
